site/content/kapp-controller/docs/v0.40.0/kctrl-package-authoring.md

```diff
@@ -428,7 +432,7 @@ the files `kapp-controller` would otherwise fetch, is available in the root of t
 
 Let's build and deploy from source.
 ```bash
-$ kctrl dev -f package-resources.yml -l -b
+$ kctrl dev -f package-resources.yml --local --build
```

Comment:
```suggestion
$ kctrl dev -f package-resources.yml --local --kbld-build
```
Flag name is `--kbld-build`. Make changes at other places also in doc

Reply:
Will get fixed when I copy stuff over from develop